pkg/cmd/run.go

```diff
@@ -134,8 +135,9 @@ func RunMicroshift(cfg *config.Config) error {
 		klog.Fatalf("MicroShift must be run privileged")
 	}
 
-	klog.InfoS("MICROSHIFT STARTING")
 	microshiftStart := time.Now()
```

Comment:
This is used by 
```
klog.InfoS("MICROSHIFT STARTING SERVICES", "since-start", time.Since(microshiftStart))
```

Do you think we could move that log to startup recorder and also record duration from "start" to "services starting"?
We could also change the signature of these 3 to not take the `time.Time`

Reply:
Added. could be nice to have, even though the time difference might  be too small to reasonably plot along the startup times of services

Reply:
Signatures changed